micropython/README.md

```diff
@@ -28,4 +28,4 @@ For building micropython we are using `unix` port provided by Micropython that i
      * `07_os_Makefile.patch`- disabling `Werror` compilation flag and addition of arithmetic functions absent in `libphoenix`
      * `08_os_varia.patch` - missing defines corrections 
      * `09_py_ifdefs.patch` - math related functions missing in `libphoenix` substitution in MicroPython code
-
+  * `exp_prefabs` - prefabricated files with expected output to some micropython tests. For most tests such files are generated using Python 3.10 or there are provided by authors of micropython. Unfortunately there are some cases in which micropython 1.15 is incompatible with newer versions of CPython. For these tests expected outputs were generated manually using micropython 1.15 installed on standard Unix system
```

Comment:
```suggestion
  * `exp_prefabs` - prefabricated files with expected output to some micropython tests. For most tests such files are generated using Python 3.10 or are provided by authors of micropython. Unfortunately there are some cases in which micropython 1.15 is incompatible with newer versions of CPython. For these tests expected outputs were generated manually using micropython 1.15 installed on standard Unix system
```
`there` unnecessary. You can use `they`. However, just an unimportant nitpick :)